tests/test-modules-export.php

```diff
@@ -140,6 +141,117 @@ public function test_getExportFolder() {
 		$this->assertStringStartsWith( 'deny from all', file_get_contents( $path . '.htaccess' ) );
 	}
 
+	public function test_getLatestExportStylePath() {
+
+		$this->_book();
+
+		$i = $this->export;
+
+		$css = '/* Silence is golden. */';
+
+		$css_files = [];
+
+		$timestamp1 = time();
+		$css_file1 = Container::get( 'Sass' )->pathToUserGeneratedCss() . "/prince-$timestamp1.css";
+		\Pressbooks\Utility\put_contents( $css_file1, $css );
+		$css_files[] = $css_file1;
+
+		$timestamp2 = time();
+		$css_file2 = Container::get( 'Sass' )->pathToUserGeneratedCss() . "/prince-$timestamp2.css";
+		\Pressbooks\Utility\put_contents( $css_file1, $css );
+		$css_files[] = $css_file2;
+
+		$latest = $i->getLatestExportStylePath( 'prince' );
+		$this->assertEquals( Container::get( 'Sass' )->pathToUserGeneratedCss() . '/prince-' . $timestamp2 . '.css', $latest );
+
+		foreach ( $css_files as $file ) {
+			if ( file_exists( $file ) ) {
+				unlink( $file );
+			}
+		}
+	}
+
+	public function test_getLatestExportStyleUrl() {
+
+		$this->_book();
+
+		$i = $this->export;
+
+		$css = '/* Silence is golden. */';
+
+		$css_files = [];
+
+		$timestamp1 = time();
+		$css_file1 = Container::get( 'Sass' )->pathToUserGeneratedCss() . "/prince-$timestamp1.css";
+		\Pressbooks\Utility\put_contents( $css_file1, $css );
+		$css_files[] = $css_file1;
+
+		$timestamp2 = time();
+		$css_file2 = Container::get( 'Sass' )->pathToUserGeneratedCss() . "/prince-$timestamp2.css";
+		\Pressbooks\Utility\put_contents( $css_file2, $css );
+		$css_files[] = $css_file2;
+
+		$latest = $i->getLatestExportStyleUrl( 'prince' );
+		$this->assertEquals( network_home_url( sprintf( '/wp-content/uploads/sites/%d/pressbooks/css/prince-%d.css', get_current_blog_id(), $timestamp2 ) ), $latest );
+
+		foreach ( $css_files as $file ) {
+			if ( file_exists( $file ) ) {
+				unlink( $file );
+			}
+		}
```

Comment:
Let's make this code more defensive so that invocations like:

```
$latest = $i->getLatestExportStyleUrl( 'haha/' )
$latest = $i->getLatestExportStyleUrl( '*+' )
$latest = $i->getLatestExportStyleUrl( '.*' )
```

Don't cause havoc?